### ISSUE TITLE
The rows stat on the Html Format of the Info tab is being divided in base 2 instead of base 10.

### ISSUE THREAD

**Issue by atulwy** [webyog/sqlyog-community#1708]
Original [issue 1708](https://code.google.com/p/sqlyog/issues/detail?id=1708) created by webyog on 2011-09-01T18:12:23.000Z:

<b>What steps will reproduce the problem?</b>
1. select a database
2. click the Info tab
3. compare the Rows stats in the Html &amp; Text/Detailed formats

<b>What is the expected output? What do you see instead?</b>
11078501 in the Text/Detailed format is showing as 10.57M when it should show 11.07M

<b>What version of the product are you using? On what operating system?</b>
Using v9.20 Beta1 on Windows XP Service Pack 3

**Comment by dehrk** [comment]
Additionally, the abbreviated row counts are using the wrong suffixes. 1145045242 rows is getting translated to 1.07G.

**Comment by peterwy** [comment]
Agreed.  There is no "bit" nor "byte" involved here. iIt is a plain number/integer ("1145045242 rows"). This should use base-10 number system. Standards' ambiguity and different historical practices for conversion of bit/byte does not apply to such plain numbers.

A low-priority fix, though.

**Comment by wyaadarsh** [comment]
We will be fixing it in the next release
